packages/logger/src/tests/logger.ts

```diff
@@ -0,0 +1,44 @@
+import { NSlogger } from '..';
+import { lorem } from 'faker';
+
+const mockDate = new Date(0);
+jest.spyOn(global, 'Date').mockImplementation(() => mockDate as any); // TypeScript workaround
+const consoleSpy = jest.spyOn((console as any)._stderr, 'write').mockImplementation();
+
+const message = lorem.sentence();
+const params: Record<string, unknown>[] = [
+  { [lorem.word()]: { [lorem.word()]: lorem.sentence() }, [lorem.word()]: lorem.sentence() },
+  { [lorem.word()]: { [lorem.word()]: lorem.sentence() } },
+];
+
+const logger = NSlogger('test');
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+afterAll(() => {
+  jest.restoreAllMocks();
+});
```

Comment:
Still wondering why this would still be here? Every test suite is fully isolated within jest so shouldn't be clean between test files unless I'm mistaken. Why would you need to restore these afterAll?